### PR TITLE
Implementation of genres lateral menu

### PR DESCRIPTION
**Pull request by FranciscoGaraventa**
*Changes of last PR:
- Renamed variables in class Dimension that not were in camel case.

I attach an image of movie card information and a gif that show how to go to that.
image

genresMenu

### CODE REVIEW

File: lib/src/widgets/backdrop_image.dart [hunk 1]
```diff
@@ -43,3 +45,12 @@ class BackdropImage extends StatelessWidget {
     return _buildContent();
   }
 }
+
```
Comment (luisjavierglb) [inline lib/src/widgets/backdrop_image.dart:48]: lets remove this comment

File: pubspec.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@ dependencies:
     sdk: flutter
   http: ^0.12.2
   flutter_staggered_grid_view: ^0.3.3
-
+  rxdart: ^0.25.0
```
Comment (luisjavierglb) [inline pubspec.yaml:28]: Im not really sure if we are using this on the project, can we remove it ?

**Review by luisjavierglb — APPROVED** [review]
LGTM

File: pubspec.yaml (hunk 2 above)
Comment (FranciscoGaraventa) [reply pubspec.yaml:28]: I need to use it in genres_bloc.

File: lib/src/widgets/backdrop_image.dart (hunk 1 above)
Comment (FranciscoGaraventa) [reply lib/src/widgets/backdrop_image.dart:48]: comment removed

File: lib/src/bloc/genres_bloc.dart [hunk 3]
```diff
@@ -1,22 +1,23 @@
+import 'package:rxdart/rxdart.dart';
 import 'bloc.dart';
 import 'dart:async';
 import '../models/genre_model.dart';
 import '../resources/movies_repository.dart';
 
 class GenresBloc extends Bloc {
   final _repository = MoviesRepository();
-  final _genresFetcherController = StreamController<GenreModel>();
+  final StreamController<GenreModel> _controller = BehaviorSubject();
```
Comment (MarcosCGlobant) [inline lib/src/bloc/genres_bloc.dart]: you should return the stream controller name to the first one, it was more descriptive

Comment (FranciscoGaraventa) [reply lib/src/bloc/genres_bloc.dart]: Done.

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**